bec_alerts/migrations/0005_auto_20180530_1444.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.4 on 2018-05-30 19:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bec_alerts', '0004_auto_20180525_1655'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='issue',
+            name='message',
+            field=models.TextField(default='', max_length=255),
+        ),
+    ]
```

Comment:
If I ever write a migration system, I'll do it with an API that causes all migrations to also be poems. Then you can take all the migrations for an app and publish it as a book of poetry and BAM! ALTERNATIVE INCOME STREAM!